tests/spread/testcraft/test-cmd/task.yaml

```diff
@@ -13,4 +13,5 @@ execute: |
   testcraft test other:...:tests/spread/my-suite/my-task $CMDLINE 2>&1 | MATCH "No matches"
 
 restore: |
+  rm -f *.testcraft
```

Comment:
Probably makes mare sense to put `rm -f *.testcraft` in spread.yaml's `restore-each`